source/user-manual/elasticsearch/elastic-tuning/configure-indices.rst

```diff
@@ -142,7 +142,7 @@ Let's suppose that we want to add a new index pattern (``my-custom-alerts-*``) a
 
```

Comment:
- Review this section and update it for the new Wazuh components: Wazuh indexer and Wazuh dashboard. 
![image](https://user-images.githubusercontent.com/61882981/164475692-90608f30-2138-4691-8bec-c35e8175065d.png)